demos/magic_distillation_demo.ipynb

```diff
@@ -1,368 +1,360 @@
 {
-    "cells": [
```

Comment:
Not sure why the whole file changed here.
Could you explain and/or show what has changed?

If possible it's preferable to only commit lines that include the _relevant_ changes.

Reply:
Oops! My bad! Will fix this :)